doc/source/conf.py

```diff
@@ -129,13 +129,13 @@
 # html_theme_options = {}
 html_theme_options = {
     'logo': 'logo.png',
-    'github_user': 'metacloud',
+    'github_user': 'ansible',
     'github_repo': 'molecule',
     'github_button': True,
     'travis_button': False,
     'show_powered_by': False,
     'extra_nav_links': {
-        'View on github': 'https://github.com/metacloud/molecule',
+        'View on GitHub': 'https://github.com/ansible/molecule',
```

Comment:
Maybe avoid hardcoding?
```suggestion
        'View on GitHub': '/'.join(('https://github.com', html_theme_options['github_user'], html_theme_options['github_repo'])),
```

Reply:
I think that's over-engineering :)

Reply:
OK, we can look at this in a future PR